README.md

```diff
@@ -122,9 +122,9 @@ You are able to execute command functions without any argument validation. If yo
 import webExt from 'web-ext';
 
 webExt.cmd.run({
+  firefox: '/path/to/Firefox-executable',
```

Comment:
Nit, let's revert this unrelated change, this line was right above `sourceDir: '...'`, at line 127.